frontend/test/pytest/test_template.py

```diff
@@ -11,17 +11,40 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Tests for QJIT compatibility of PennyLane templates."""
 
 import jax
 import jax.numpy as jnp
 import numpy as np
 import pennylane as qml
 import pytest
-from pennylane import numpy as pnp
 from scipy.stats import norm
 
 from catalyst import for_loop, qjit
 
+# pylint: disable=too-many-lines
+
```

Comment:
Please remove the extra blank line unless black wants it.

Reply:
Black wants this line.